packages/web/src/pages/private-key-exporter-page/PrivateKeyExporterPage.tsx

```diff
@@ -0,0 +1,385 @@
+      <Text variant='heading' css={{ fontSize: 34 }} color='accent'>
+        {messages.showPrivateKey}
+      </Text>
+      <Flex gap='s'>
+        <IconTriangleExclamation fill={color.special.darkRed} />
+        <Text variant='body' color='danger' size='l' strength='strong'>
+          {messages.yourOwnRisk}
+        </Text>
+      </Flex>
+    </Flex>
+  )
+}
+
+const WhatDoesThisDo = () => {
+  const { color } = useTheme()
+  return (
+    <Flex direction='column' alignItems='flex-start' mv='l' pv='xl' gap='l'>
+      <Flex alignItems='center' gap='s'>
+        <IconQuestionCircle size='m' fill={color.neutral.n800} />
+        <Text variant='title' css={{ fontSize: 24 }}>
+          {messages.what}
+        </Text>
+      </Flex>
+      <Text variant='body' textAlign='left'>
+        {messages.thisTool}
+      </Text>
+      <Text variant='body' textAlign='left'>
+        {messages.advancedUsers}
+      </Text>
+      <Text variant='body' textAlign='left' strength='strong'>
+        {messages.note}
+      </Text>
+    </Flex>
+  )
+}
+
+const UnderstandTheRisks = () => {
+  const { color } = useTheme()
+  const bulletPoints = [
+    messages.untrustworthyParties,
+    messages.compromisingSecurity,
+    messages.lossOfFunds,
+    messages.unforeseenConsequences
+  ]
+  return (
+    <Box pv='xl'>
+      <Flex
+        direction='column'
+        alignItems='flex-start'
+        pv='l'
+        ph='2xl'
+        gap='l'
+        borderRadius='m'
+        border='strong'
+        css={{
+          background: 'rgba(208, 2, 27, 0.05)',
+          borderColor: color.special.darkRed
+        }}
+      >
+        <Flex alignItems='center' gap='s'>
+          <IconTriangleExclamation size='m' fill={color.special.darkRed} />
+          <Text variant='title' color='danger' css={{ fontSize: 24 }}>
+            {messages.understandTheRisk}
+          </Text>
+        </Flex>
+        <Text variant='body' strength='strong' textAlign='left' color='danger'>
+          {messages.improperUse}
+        </Text>
+        {bulletPoints.map((bulletPoint, i) => (
+          <Text
+            key={`bullet-point${i}`}
+            variant='body'
+            textAlign='left'
+            color='danger'
+            css={{
+              display: 'list-item',
+              listStyleType: 'disc',
+              listStylePosition: 'inside'
+            }}
+          >
+            {bulletPoint}
+          </Text>
+        ))}
+      </Flex>
+    </Box>
+  )
+}
+
+const KeepItSecret = () => {
+  const { color } = useTheme()
+  const bulletPoints = [
+    messages.yourPrivateKey,
+    messages.audiusWillNever,
+    messages.avoidPhishing
+  ]
+  return (
+    <Flex direction='column' alignItems='flex-start' mv='l' pv='xl' gap='l'>
+      <Flex alignItems='center' gap='s'>
+        <IconVisibilityHidden size='m' fill={color.neutral.n800} />
+        <Text variant='title' css={{ fontSize: 24 }}>
```

Comment:
Figma says this is heading/medium/regular?

Reply:
thx. updated